config/routes.rb

```diff
@@ -10,7 +10,10 @@
   end
 
   resources :groups, only: %i(index create destroy)
-  resources :appointments
+  resources :appointment_attempts do
+    resources :appointments
+  end
+  get '/not_eligible', to: 'appointment_attempts#not_eligible'
```

Comment:
This should be moved into the `:appointment_attempts` resources declaration. Actually, it's worth checking but I think if you have a template with that name, you don't even need this routing declaration as they get rendered implicitly.